### ISSUE TITLE
Investigate outbound Twilio possibilities

### ISSUE THREAD

**Issue by mc-funk** [Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations#85]
There are two use cases for outbound texts that the TCMAP team wants to look into: 
1. being able to respond to folks who send us information. Not sure what this looks like on the process side, but if you can let us know the possibilities, we can get that figured out.

2. Stretch goal: being able to send out reminders, preferably automated, to sites that have subscribed. 
  - our vision is to send out a rotating set of reminders no more than daily, which include self-care messages such as "Have you drank water recently? Also, please remember to update us on your site's needs and hours. Thanks for all you do!" (we can get a list of these that are better than that). This is inspired by our reminders in the TCMAP #love-and-support channel which we lovingly call "Robot Dad". - This would need all the standard opt out options or we would need to select who gets them so that we're not spamming people who don't want them. I'm not sure how that fits in to all our various considerations e.g. budget, feasibility, whether twilio is the appropriate tool for this.

**Comment by maxine** [comment]
@mc-funk after some investigation, I have an Apps Script all set up so that you can send messages from the Twilio phone number by putting the message in Google Sheets ([based on this tutorial](https://www.twilio.com/blog/2016/02/send-sms-from-a-google-spreadsheet.html)). I can put that in the main Google Spreadsheet whenever! This can be used for reminders or for direct replies to people (and I can set those two spreadsheets up separately).

**Comment by mc-funk** [comment]
Wow, that's sooooo cool!! I will surface that back to the group, it'll require some training and decision-making on the spreadsheet side of things, but awesome potential and I really appreciate it!!!

**Comment by andyinabox** [comment]
@maxine can we add the script to https://github.com/Twin-Cities-Mutual-Aid/community-map-google-scripts ?

**Comment by maxine** [comment]
@andyinabox good call! See [PR here](https://github.com/Twin-Cities-Mutual-Aid/community-map-google-scripts/pull/1).

**Comment by maxine** [comment]
Can also add anyone to the Twilio account and/or share login info

**Comment by maxine** [comment]
@mc-funk @comeoneileen [created a PR ](https://github.com/Twin-Cities-Mutual-Aid/community-map-google-scripts/pull/3) describing changes to enable offline auto-responses in the `community-map-google-scripts` repository. Hopping offline for an hour or so, but let me know what you think when you have a chance and I can make any adjustments today before adding it to the spreadsheet.
